cmd/controller/main.go

```diff
@@ -114,15 +112,17 @@ func main() {
 	recorder := events.NewDedupeRecorder(events.NewRecorder(manager.GetEventRecorderFor(appName)))
 	cluster := state.NewCluster(manager.GetClient(), cloudProvider)
 
+	metrics.NewMetricCollector(ctx, cluster)
```

Comment:
This does return a `MetricScraper` instance but since it isn't used anywhere at the moment it couldn't be assigned. One of the potential changes I talked about in a different comment, updating the metrics at the end of existing reconcile requests, would make use of this. If I don't go forward with that though would it be more idiomatic to not return the instance at all and just call it `StartMetricScraper`?